week-1/E-strings/exercise.js

```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
 
+//Program that logs a message and it's type
+
+var message = 'This is a string';
+var messageType = typeof message;
+
 console.log(message);
+console.log(messageType);
```

Comment:
This is correct though it's identical to the example.

You could also try another type of message:

```
var num = 5;
var numType = typeof num;
console.log(num);
console.log(numType);
```

This will give the last line as `number` to show that is the type of `num`.